lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2686,7 +2687,12 @@ class HSADevice final : public KalmarDevice
             case hc::EF_AMDGPU_MACH_AMDGCN_GFX906 : triple.append("906"); break;
         }
 
-        const auto isa{get_isa_name_from_triple(std::move(triple))};
+        unsigned SRAM_ECC_ENABLED = reader.get_flags() & hc::EF_AMDGPU_SRAM_ECC;
```

Comment:
All caps is for macros, this isn't a macro.

Reply:
corrected in latest commit